tests/_support/_generated/UnitTesterActions.php

```diff
@@ -540,9 +565,45 @@ public function fail($message) {
      *
      * @param $exception string or \Exception
      * @param $callback
+     *
+     * @deprecated Use expectThrowable instead
```

Comment:
Is this valid?

Reply:
Yeah, it's generated code from codeception.